apps/omg_watcher_rpc/lib/tracer.ex

```diff
@@ -18,4 +18,26 @@ defmodule OMG.WatcherRPC.Tracer do
   """
 
   use Spandex.Tracer, otp_app: :omg_watcher_rpc
+  alias OMG.WatcherRPC.Application
+
+  def add_trace_metadata(%{assigns: %{error_type: error_type, error_msg: error_msg}} = conn) do
+    service_name = Application.service_name()
+    version = Application.version()
+
+    conn
+    |> SpandexPhoenix.default_metadata()
+    |> Keyword.put(:service, service_name)
+    |> Keyword.put(:error, [{:error, true}])
+    |> Keyword.put(:tags, [{:version, version}, {:"error.type", error_type}, {:"error.msg", error_msg}])
+  end
+
+  def add_trace_metadata(conn) do
+    service_name = Application.service_name()
```

Comment:
```suggestion
    service_name = OMG.WatcherRPC.Configuration.service_name()
    service_name = OMG.WatcherRPC.Configuration.version()
```
And if the configuration module doesn't exist, you can create one

Reply:
I've refactored these functions into a new Configuration module.